crates/nu-cli/tests/completions/mod.rs

```diff
@@ -2199,6 +2199,16 @@ fn exact_match() {
         &vec![file(dir.join("partial").join("hello.txt")).as_str()],
         &suggestions,
     );
+
+    let target_dir_2 = format!("open {}", file(dir.join("partial").join("hello"))); // No trailing slash
+    let suggestions_2 = completer.complete(&target_dir_2, target_dir_2.len());
+
+    // Despite no trailing slash, since it's an exact match, only 'partial/hello.txt' should be suggested, not
+    // 'partial-a/hello' and stuff. Implemented in #15387
+    match_suggestions(
+        &vec![file(dir.join("partial").join("hello.txt")).as_str()],
+        &suggestions_2,
```

Comment:
Rust lets you shadow variables, so you can call these `target_dir` and `suggestions` too.
```suggestion
    let target_dir = format!("open {}", file(dir.join("partial").join("hello"))); // No trailing slash
    let suggestions = completer.complete(&target_dir, target_dir.len());

    // Despite no trailing slash, since it's an exact match, only 'partial/hello.txt' should be suggested, not
    // 'partial-a/hello' and stuff. Implemented in #15387
    match_suggestions(
        &vec![file(dir.join("partial").join("hello.txt")).as_str()],
        &suggestions,
```
The comment also isn't really necessary, but it's fine to leave it in. In general, there's no need to mention which PR a test comes from. I think I just mentioned #13302 above because #13302 implemented the exact match behavior and then I later added a test in a different PR.

Reply:
If you want to crossreference, the issue id is probably more helpful to directly point to the original bug report etc.